tests/13-ieee802154/09-cooja-test-csma-security.csc

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<simconf>
```

Comment:
If you run `cooja --args=--update-simulation 09-cooja-test-csma-security.csc`, the file should be updated with the latest changes.

Reply:
If you haven't seen the behavior before: Cooja will start with the GUI, and then immediately shut down. It's not intuitive, but the file should be updated after shutting down.